### PR TITLE
jdk updated to jdk-8u121-linux-x64

### PR DESCRIPTION
**Pull request by zygimantus**

### CODE REVIEW

File: common/provision/default/image-prep.sh [hunk 1]
```diff
@@ -35,3 +35,4 @@ if [ ! -e "${provision_path}/files/${product_name}-${product_version}.zip" ]; th
 fi
 
 export file_location=$provision_path/files
+z
```
Comment (hof) [inline common/provision/default/image-prep.sh]: Is this a typo?

**Comment by hof** [comment]
Probably should also make the jdk dir in the image version independent: 

`common/provision/default/image-config.sh:jdk_install_dir=/mnt/jdk-7u80`

File: common/provision/default/image-prep.sh (hunk 1 above)
Comment (zygimantus) [inline common/provision/default/image-prep.sh]: Sorry, yes, this is a typo.

**Comment by anuruddhal** [comment]
jdk7 is recommended for these wso2servers.

**Comment by zygimantus** [comment]
As far as I know, wso2 should work just fine with jdk8.

**Review by iamvinodshabadi — COMMENTED** [review]
should we not change file - common/provision/default/image-config.sh
# JDK version
-jdk_install_dir=/mnt/jdk-7u80
+jdk_install_dir=/mnt/jdk-8u221

**Pull request closed without merging**